src/app/pages/home/home.page.scss

```diff
@@ -12,7 +12,9 @@ mat-sidenav {
 }
 
 mat-sidenav-content {
-  overflow: hidden;
+  width: 100%;
+  overflow-y: scroll;
+  overflow-x: hidden;
 }
 
 mat-toolbar {
```

Comment:
The color of `mat-toolbar` can be changed systematically by applying the `color="primary"` attribute to template from Angular Material Library. We should always prefer the styling attributions provided by Angular Material Library than plain CSS rules.